skore/src/skore/sklearn/_plot/data/table_report.py

```diff
@@ -0,0 +1,49 @@
+from skrub._reporting._html import to_html
+from skrub._reporting._summarize import summarize_dataframe
+
+from skore.sklearn._plot.base import Display
+
+
+class TableReportDisplay(Display):
+    def __init__(self, summary, column_filters=None):
+        self.summary = summary
+        self.column_filters = column_filters
+
+    @classmethod
+    def from_frame(cls, df, with_plots=True, title=None):
+        summary = summarize_dataframe(
+            df,
+            with_plots=with_plots,
+            title=title,
+        )
+        return cls(summary)
+
+    def plot(self, kind=None):
```

Comment:
To allow the `set_style` method as well:

```suggestion
    @StyleDisplayMixin.style_plot
    def plot(self, kind=None):
```